components/Guide/Guide.js

```diff
@@ -1,10 +1,25 @@
 import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+
 import Markdown from '../../components/Markdown';
 import Menu from './Menu';
 
 import './Guide.scss';
+import { Window } from '../../lib/Location';
```

Comment:
Yes. Can rename Location.js

Of wrap raw eventListener to a package. That package will do no-op on `canUseDom` is false.